externals/kyuubi-trino-engine/src/main/scala/org/apache/kyuubi/engine/trino/TrinoConf.scala

```diff
@@ -30,4 +32,11 @@ object TrinoConf {
       .version("1.5.0")
       .intConf
       .createWithDefault(3)
+
+  val CLIENT_REQUEST_TIMEOUT: ConfigEntry[Long] =
+    buildConf("trino.client.request.timeout")
+      .doc("Client request will be closed when it's not accessed for this duration of time")
```

Comment:
the `client` here is very ambiguous

Reply:
Not sure only doc or key is included, had optimized doc first.